tests/system/test_identifier.py

```diff
@@ -0,0 +1,148 @@
+import pandas as pd
+import numpy as np
+from csodiaq.identifier import Identifier
+from csodiaq.loaders.query import QueryLoaderContext
+import os
+import pickle
+import pytest
+
+
+def get_parent_dir():
+    return os.path.dirname(os.path.abspath(__file__))
+
+
+def get_file_from_system_test_folder(file):
+    return os.path.join(get_parent_dir(), "test_files", file)
+
+
+@pytest.fixture
+def commandLineArgs():
+    args = {
+        "command": "id",
+        "files": [get_file_from_system_test_folder("20190412_DI2A_1to1_tMS2_n3.mzXML")],
+        "library": get_file_from_system_test_folder("system_test_library.csv"),
+        "outDirectory": "",
+        "fragmentMassTolerance": 30,
+        "correction": 0,
+        "histogram": False,
+        "proteinTargets": 1,
+        "query": 0,
+        "heavyMz": True,
+        "peaks": 0,
+        "commonpeptide": False,
+        "commonprotein": False,
+    }
+    return args
+
+
+def assert_numeric_pandas_dataframes_are_equal(expectedDf, df, type):
+    assert len(expectedDf.index) == len(df.index)
+    columns = get_columns_that_should_match(type)
+    for columnName in columns:
+        assert columnName in df.columns
+        expectedColumn = np.array(expectedDf[columnName])
+        column = np.array(df[columnName])
+        expectedColumn.sort()
+        column.sort()
+        if expectedColumn.dtype.kind in np.typecodes["AllFloat"]:
+            np.testing.assert_array_almost_equal(expectedColumn, column)
+        else:
+            np.testing.assert_array_equal(expectedColumn, column)
+
+
+def get_columns_that_should_match(type):
+    if type == "match":
+        return [
+            "libraryIntensity",
+            "queryIntensity",
+            "ppmDifference",
+        ]
+    elif type == "score":
+        return [
+            "cosineScore",
+            "maccScore",
+        ]
+    elif type == "full":
+        return ["scan", "peptide", "cosine"]
+    elif type == "spectral":
+        return [
+            "scan",
+            "peptide",
+            "cosine",
+            "spectralFDR",
+        ]
+    elif type == "peptide":
+        return [
+            "scan",
+            "peptide",
+            "cosine",
+            "peptideFDR",
+        ]
+    elif type == "protein":
+        return [
+            "scan",
+            "peptide",
+            "cosine",
+            "leadingProtein",
+            "proteinCosine",
+            "peptideFDR",
+            "leadingProteinFDR",
+        ]
```

Comment:
Recommend delete; only use is in `assert_numeric_pandas_dataframes_are_equal`, which can be removed.